src/Common/SSHWrapper.cpp

```diff
@@ -169,9 +169,15 @@ String SSHKey::getKeyType() const
     return ssh_key_type_to_char(ssh_key_type(key));
 }
 
+void SSHKey::setNeedsDeallocation(bool needs_deallocation_)
```

Comment:
SSH key is a duplicate of src/SSH/SSHPublicKey. Further refactoring should remove this